src/librbd/ImageCtx.h

```diff
@@ -207,6 +207,7 @@ namespace librbd {
     bool clone_copy_on_read;
     bool enable_alloc_hint;
     uint32_t alloc_hint_flags = 0U;
+    uint32_t op_flags = 0U;
```

Comment:
Nit: it's an op-flag:
```suggestion
    uint32_t read_flags = 0U;
```